Updated the page the prerequisites and for Mac OS (#1036)
ColoredCow/portal: docs/prerequisites.md

## Prerequisites

For all the beginners that are starting on the portal, this is for you to get started.

### 1. Git :cat:
Git should come pre-installed in Linux and macOS. For Windows, use [this link](https://git-scm.com/download/win)
After you've successfully installed, you can verify the installation using:

```sh
git --version

> # git version 2.33.0.windows.2

Reviewer: Can we add a line `output should be something like this` ?

Author: Okay

## Prerequisites

For all the beginners that are starting on the portal, this is for you to get started.

### 1. Git :cat:

> For Windows only:
>   - [this link](https://git-scm.com/download/win)
>     Linux and MacOS comes pre installed

Reviewer 2: ```suggestion
Git should come pre-installed in Linux and macOS. For Windows, use [this link](https://git-scm.com/download/win).
```

After you've successfully installed, you can verify the installation using:

>   - ```sh
>       git --version
>     # git version 2.33.0.windows.2

Reviewer 2: ```suggestion
```sh
git --version
# git version 2.33.0.windows.2
```

    ```

> ### 2. PHP,MYSQL,APACHE :running:

Reviewer 2: ```suggestion
### 2. PHP, MySQL, Apache :running:
```

> Portal uses PHP v7.3 or v7.4 is needed. You can install it with the following ways:

Reviewer 2: ```suggestion
Portal uses PHP v7.4. You can install it using one of the following ways:
```

      composer update
      ```
   - If the user is working in VSCode, possible errors may arise because of path, as may not be able to access the selected PHP version from its terminal.

> 3. MAMP (Windows and MacOS)

Reviewer 2: ```suggestion
3. MAMP (macOS)
```

    - If you prefer using MAMP, you can download with all the PHP version from [this link](https://www.mamp.info/en/downloads/)this link for Windows and MacOS.
    -To use the MAMP php as main php, refer[this link](https://stackoverflow.com/questions/4262006/how-to-use-mamps-version-of-php-instead-of-the-default-on-osx) for switching into the MAMP php.
4. After you've successfully installed, you can verify the installation using:

>    - ```sh
>       php -v
>       # output should be something like
>       # PHP 7.4.21 (cli) (built: Aug  5 2021 15:34:00) ( NTS )
>       #Copyright (c) The PHP Group
>       #Zend Engine v3.4.0, Copyright (c) Zend Technologies   

Reviewer 2: ```suggestion
```sh
php -v
# output should be something like
# PHP 7.4.21 (cli) (built: Aug  5 2021 15:34:00) ( NTS )
```

      ```   

### 3. Composer :running:

For MacOS:
- [Composer v2.0](https://getcomposer.org/download)

For Windows:
- [Composer v2.0](https://getcomposer.org/doc/00-intro.md#installation-windows)

For Linux:
- [Composer v2.0](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-macos)

After you've successfully installed, you can verify the installation using:
```sh
composer --version
# output should be something like
# Composer version 2.1.6 2021-08-19 17:11:08
```

### 4. Nodejs :running:

> For (Windows, Ubuntu, MacOS)

Reviewer 2: ```suggestion
For (Windows, Ubuntu, macOS)
```

  - [Laravel Blade Snippets for Blade files](https://marketplace.visualstudio.com/items?itemName=onecentlin.laravel-blade)
  - [Prettier - for JS, CSS, HTML files](https://prettier.io/)

> ### 6.Automated Testing :computer:

Reviewer 2: remove

  - [Laravel Blade Snippets for Blade files](https://marketplace.visualstudio.com/items?itemName=onecentlin.laravel-blade)
  - [Prettier - for JS, CSS, HTML files](https://prettier.io/)

> ### 8. Automated Testing :computer:

Reviewer 2: why removed this?

Reviewer: @gautam071 I see this comment is unresolved, please check
